pkgs/nix-tools/darwin-rebuild.sh

```diff
@@ -127,6 +131,11 @@ if [ -z "$action" ]; then showSyntax; fi
 
 flakeFlags=(--extra-experimental-features 'nix-command flakes')
 
+# Use ~/.config/darwin/flake.nix if it exists. It can be a symlink to the actual flake.
+if [[ -z $flake && -e ~/.config/darwin/flake.nix && -z $noFlake ]]; then
+    flake="$(dirname "$(readlink -f ~/.config/darwin/flake.nix)")"
```

Comment:
```suggestion
# Use /etc/nix-darwin/flake.nix if it exists. It can be a symlink to the actual flake.
if [[ -z $flake && -e /etc/nix-darwin/flake.nix && -z $noFlake ]]; then
    flake="$(dirname "$(readlink -f /etc/nix-darwin/flake.nix)")"
```